conda_smithy/lint_recipe.py

```diff
@@ -413,6 +413,25 @@ def _maintainer_exists(maintainer: str) -> bool:
         )
 
 
+def _team_exists(org_team: str) -> bool:
+    """Check if a team exists on GitHub."""
+    if "GH_TOKEN" in os.environ:
+        _res = org_team.split("/", 1)
+        if len(_res) != 2:
+            return False
+        org, team = _res
+        gh = _cached_gh()
+        _org = gh.get_organization(org)
```

Comment:
Can we cache this too?

Reply:
https://github.com/conda-forge/conda-smithy/pull/2093